docs/getting-started/with-python.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Quick Start With Python Client SDK
```

Comment:
The title "Quick Start With Python" could be enough (same for the other languages)